Update browser module dblclick docs (#1456)
grafana/k6-docs: docs/sources/next/javascript-api/k6-experimental/browser/frame/dblclick.md

---
title: 'dblclick(selector[, options])'
excerpt: 'Browser module: frame.dblclick(selector[, options]) method'
---

# dblclick(selector[, options])

{{% admonition type="warning" %}}

Use locator-based [`locator.dblclick([options])`](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/browser/locator/dblclick/) instead.

{{% /admonition %}}

> Double clicks an element matching provided selector.

Reviewer: ```suggestion
Double clicks on an element matching the provided selector.
```

<TableWithNestedRows>

| Parameter           | Type     | Default | Description                                                                                                                                                                                                                                                                                                                                   |
| ------------------- | -------- | ------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |

> | selector            | string   | `''`    | A selector to search for an element. If there are multiple elements satisfying the selector, the first will be used.                                                                                                                                                                                                                          |

Reviewer: ```suggestion
| selector            | string   | `''`    | A selector to search for an element. If multiple elements match the selector, the first element will be used.  
```

| options             | object   | `null`  |                                                                                                                                                                                                                                                                                                                                               |
| options.button      | string   | `left`  | The mouse button (`left`, `middle` or `right`) to use during the action.                                                                                                                                                                                                                                                                      |
| options.delay       | number   | `0`     | Milliseconds to wait between `mousedown` and `mouseup`.                                                                                                                                                                                                                                                                                       |

> | options.force       | boolean  | `false` | Bypasses the actionability checks (i.e., `visible`, `stable`, `enabled`) if set to `true`. 

Reviewer 2: Same as previous comment.

> | options.force       | boolean  | `false` | Setting this to `true` will bypass the actionability checks (`visible`, `stable`, `enabled`).                                                                                                                                                                                                                                                 |

Reviewer: ```suggestion
| options.force       | boolean  | `false` | Bypasses the actionability checks (i.e., `visible`, `stable`, `enabled`) if set to `true`. 
```
